ingester/ingester.go

```diff
@@ -50,14 +50,14 @@ type FirehoseIngester struct {
 }
 
 func NewFirehoseIngester(
-	log *zap.Logger, queries *store.Queries, crc *CandidateActorCache,
+	log *zap.Logger, queries *store.Queries, crc *CandidateActorCache, subscribeURL string,
 ) *FirehoseIngester {
 	return &FirehoseIngester{
 		log:     log,
 		crc:     crc,
 		queries: queries,
 
-		subscribeURL:    "wss://bsky.social/xrpc/com.atproto.sync.subscribeRepos",
+		subscribeURL:    subscribeURL,
```

Comment:
Nit: Perhaps we make this `bgsHost` or `pdsHost` rather than injecting the full URL?

Reply:
oh yeah, this now takes in the `pdsURL` :+1: